frontend/package.json

```diff
@@ -14,6 +14,7 @@
     "test": "vitest",
     "test:e2e": "playwright test",
     "format:check": "prettier --cache --check .",
+    "format:write": "prettier --write .",
```

Comment:
I don't think we need this and might remove it in a future commit. To fix formatting issues all you have to do is run `npm format:check -- --write`.

Reply:
Will resolve this conversation.. it's not a blocker.